### ISSUE TITLE
Error on creating countersheet

### ISSUE THREAD

**Issue by FliesWithCrows** [lifelike/countersheetsextension#73]
Possibly user error,  but I have just updated inkscape and the countersheet extension. I am getting the following error when i attempt to import data - (the CSV and SVG worked in the previous version).  

  File "countersheet.py", line 1092
    line.set("style", self.find_registration_line_style())
                                                         ^
TabError: inconsistent use of tabs and spaces in indentation

**Comment by lifelike** [comment]
Are you sure you got the correct versions? That sounds like an error message you would get if you run an older version of the extension using Inkscape 1.x? The last tabs I am aware of were removed in commit 81952b9c3c53d0c88ba26485acbbfeebf9c39646 in February 2020, so 3.0 (https://github.com/lifelike/countersheetsextension/releases/tag/3.0) should not have that issue.

**Comment by FliesWithCrows** [comment]
Thank you for suggesting. So that's what i thought so i copy/pasted the extension again using the link you provided. No joy. 
I then uninstalled inkscape (deleting any remaining files) and then reinstalled inkscape and the extension, but still the same error.

**Comment by FliesWithCrows** [comment]
Somewhere along the line I resolved the issue. 
- I uninstalled inkscape. I searched for all files relating to inkscape or countersheet on the computer and deleted them. 
- Then reinstalled inkscape without extensions and then installed countersheet - got a different error
- Uninstalled inkscape, deleted all files relating to countersheet
- Installed inkscape with extensions, installed countersheet extension. 
Error has gone. 
(Although the extension is importing pngs in a way that is unpredictable and seems to defy logic)
